atom/browser/api/atom_api_app.cc

```diff
@@ -874,6 +875,50 @@ std::string App::GetLocale() {
   return g_browser_process->GetApplicationLocale();
 }
 
+std::string App::GetRegion() {
+  std::string region;
+#if defined(OS_WIN)
+  WCHAR locale_name[LOCALE_NAME_MAX_LENGTH] = {0};
+
+  if (GetLocaleInfoEx(LOCALE_NAME_USER_DEFAULT, LOCALE_SISO3166CTRYNAME,
+                      (LPWSTR)&locale_name,
+                      sizeof(locale_name) / sizeof(WCHAR)) ||
+      GetLocaleInfoEx(LOCALE_NAME_SYSTEM_DEFAULT, LOCALE_SISO3166CTRYNAME,
+                      (LPWSTR)&locale_name,
+                      sizeof(locale_name) / sizeof(WCHAR)))
+    base::WideToUTF8(locale_name, wcslen(locale_name), &region);
+#elif defined(OS_MACOSX)
+  CFLocaleRef locale = CFLocaleCopyCurrent();
+  CFStringRef value = CFStringRef(
+      static_cast<CFTypeRef>(CFLocaleGetValue(locale, kCFLocaleCountryCode)));
+  const CFIndex kCStringSize = 128;
+  char temporaryCString[kCStringSize];
+  bzero(temporaryCString, kCStringSize);
```

Comment:
[`bzero()`](http://pubs.opengroup.org/onlinepubs/009696699/functions/bzero.html) is legacy; prefer [`memset()`](http://pubs.opengroup.org/onlinepubs/009696699/functions/memset.html)

Reply:
or just `char temporaryCString[kCStringSize] = {}`